cogs/cog_easteregg.py

```diff
@@ -0,0 +1,218 @@
+import discord
+import asyncio
+from discord.ext import commands
+import random
+from utils import config as cfg
+
+
+class EasterEggCommands(commands.Cog, name="Easter Egg Commands"):
+    """These are the easter egg commands"""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="8-ball", aliases=["8ball", "8b"])  # command to create a note
+    async def eight_ball(self, ctx, *, question: str):
+        """ Play magic 8-ball, answers to everything """
+        try:
+            positive_responses = [
+                'It is certain.',
+                'It is decidedly so.',
+                'Without a doubt',
+                'Yes - definetly',
+                'You may rely on it.',
+                'As I see it, yes.',
+                'Most likely.',
+                'Outlook good.',
+                'Yes.',
+                'Signs point to yes.'
+            ]
+            hazy_responses = [
+                'Reply hazy, try again',
+                'Ask again later.',
+                'Better not tell you now.',
+                'Cannot predict now',
+                'Concentrate and ask again.'
+            ]
+            negative_responses = [
+                'My reply is no.',
+                'My sources say no.',
+                'Outlook not so good.',
+                'Very doubtful.'
+            ]
+            responses = [*positive_responses, *hazy_responses, *negative_responses] # single list of responses
+            response = random.choice(responses) # select response
+
+            # determine type of response for embed
+            if response in positive_responses:
+                colour = cfg.colors.SUCCESS
+            elif response in hazy_responses:
+                colour = cfg.colors.WSU_GOLD
+            elif response in negative_responses:
+                colour = cfg.colors.ERROR
+
+            # output embed
+            embed = discord.Embed(title="Magic 8-Ball:", color=colour)
+            embed.set_author(name=ctx.author.display_name, icon_url=ctx.author.avatar_url)
+            embed.set_thumbnail(
+                url="https://external-content.duckduckgo.com/iu/" +
+                    "?u=https%3A%2F%2Fwww.bigw.com.au%2Fmedias%2Fsys" +
+                    "_master%2Fimages%2Fimages%2Fhd5%2Fh14%2F13660536602654.jpg&f=1&nofb=1")
+            embed.add_field(name=response, value="Statement: " + question, inline=True)
+            await ctx.send(embed=embed)
```

Comment:
Does this display the embed to the user? Should add comment.

Reply:
I understand, I will clear it up